lib/testing/src/lib/core/pages/login.page.ts

```diff
@@ -50,7 +65,12 @@ export class LoginPage {
     logoSwitch = element(by.id('adf-toggle-logo'));
```

Comment:
demo-shell specific, should not be in the generic page object

Reply:
besides the other ones I added comments to, the following are also demo-shell specific and should not be in the generic page object:
footerSwitch
rememberMeSwitch
successRouteSwitch